README.md

```diff
@@ -0,0 +1,32 @@
+## How to Make Changes to the Group Website
+
+### A. From your own forked repo
+
+#### I. Getting Started
+
+1. Fork the original [repo](https://github.com/az-digitalag/group-website)
+2. If you do not already have RStudio installed, follow these [instructions](https://rstudio.com/products/rstudio/download/)
+3. Open RStudio and navigate to your forked repo
+4. If not already downloaded, install the blogdown and hugo packages using these [instructions](https://bookdown.org/yihui/blogdown/installation.html)
+5. To preview the website locally, use the command `blogdown::serve_site()` in the console
+6. You should now be able to preview the group website within RStudio, or copy and paste the url provided in the console into your browser. 
+
+#### II. Keeping your forked repo up to date with new changes from the command line
```

Comment:
Can this just reference / be a link to our group wiki in the section on our standard / generic git workflow?

Reply:
Good idea!